### DISCUSSION TITLE
Revise release process

### DISCUSSION THREAD

**Discussion by mnriem** [piranhacloud/piranha#1976] [Roadmap]
Looking at the current release process it can hold up contributions as we currently have to freeze the branch the release is cut from for commits for a longer time while we collect the TCK numbers. I think we should adopt a different model and would like to suggest we go to a process where a release branch is created and the release is done from there.

Thoughts?

**Comment by Thihup** [comment]
Sounds great. I'm only concerned if the branch creation can be automated, otherwise, it can be more work for you to do in each monthly release.

**Reply by mnriem** [reply]
I hear you, but I also do not want to block any contributions. I think cutting a release branch on the 12th  and then validating the TCKs against that branch to get the numbers adds a bit of overhead for the release but it means that contributions can keep coming ;)
